### PR TITLE
update dependencies for flutter 3.10

### PR DESCRIPTION
**Pull request by BentEngbers**
*Replace this paragraph with a description of PR*

## Pre-launch Checklist

- [x] I updated `pubspec.yaml` with an appropriate new version.
- [x] I updated `CHANGELOG.md` with new version number and description.
- [x] I updated `README.md`.
- [x] I updated example project with new feature.
- [x] Check if the branch name is correct.
- [x] Code is warning free.

### CODE REVIEW

**Comment by kyryd** [comment]
Hi there,
We are struggling with migration to flutter 3.10.
Is there any chance that the pull request could be approved and merged?

**Comment by kyryd** [comment]
> > Hi there,
> > We are struggling with migration to flutter 3.10.
> > Is there any chance that the pull request could be approved and merged?
> 
> Sure. I am gonna take action today 👍

Great! 
Thanks a lot !

**Comment by BentEngbers** [comment]
> @BentEngbers can you update the version number in pubspec.yaml and README.md file s?

Done!

File: pubspec.yaml [hunk 1]
```diff
@@ -10,14 +10,14 @@ environment:
...
-  intl: ^0.17.0
+  intl: ^0.18.0
   plugin_platform_interface: ^2.0.2
   universal_io: ^2.0.4
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  flutter_lints: ^1.0.4
+  flutter_lints: ^2.0.1
```
Comment (uygar) [inline pubspec.yaml:20]: Can we downgrade this version? Additionally, we want to continue supporting Flutter 2.10.5

**Review by uygar — CHANGES_REQUESTED** [review]

File: pubspec.yaml (hunk 1 above)
Comment (BentEngbers) [reply pubspec.yaml]: Ive downgraded the flutter_lints dependency as requested. 
For the new flutter version, the minimal version is 2.12.0. 
CleanShot 2023-05-18 at 23 03 14@2x

**Review by uygar — APPROVED** [review]

**Pull request merged**